src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Declares function templates to calculate the Ricci tensor
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic {
+
+// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute spatial Ricci tensor using evolved variables and
+ * their first derivatives.
+ *
+ * \details Compute \f$R_{ij}\f$ as:
+ *
+ * \f{align}
+ * R_{i j} =& \case{1}{2} g^{a b} \left( \partial_{(i} d_{a b j)}
```

Comment:
I think the braces are off here as well -- the `d_{a b j}` at the end of the line will end up in the partial subscript as written.
(regarding Geoffrey's comment - `case` didn't render for me in doxygen, so I also think you should just go with `frac`)

Reply:
yeah, fixed that and `case` should have been `frac` (autotyping error).